src/MasterSite.php

```diff
@@ -655,6 +655,7 @@ function ($item) {
         // or enforce_cookies_policy setting is not enabled.
         $context['enforce_cookies_policy'] = isset($options['enforce_cookies_policy']) ? true : false;
         $context['google_tag_value'] = $options['google_tag_manager_identifier'] ?? '';
+        $context['google_tag_domain'] = !empty($options['google_tag_manager_domain']) ? $options['google_tag_manager_domain'] : 'www.googletagmanager.com';
```

Comment:
Can you break up this line for readability? `phpcs` linting tests fail:

```
Line exceeds maximum limit of 120 characters; contains 155 characters
```

Reply:
Done!